### DISCUSSION TITLE
Change clip distance for anchors

### DISCUSSION THREAD

**Discussion by gsechan** [sceneview/sceneview#423] [Q&A]
It looks like with the geospatial API that anchors beyond around 200m clip out of view.  I'd like to increase that to 400m.  What property controls that?  I found some hints for old sceneform, but nothing for sceneview.

**Comment by gsechan** [comment]
And as seems to be my pattern, I find it 20 minutes after I post here.  arView.cameraNode.far = value will set the max render distance and arView.cameraNode.near = value will set the minimum

**Comment by ThomasGorisse** [comment]
In v4.0.0-rc.1, the camera's near/far clip planes can be configured through the Filament camera settings. In AR mode, ARCore controls the projection matrix to match the physical camera, but you can adjust the near/far clip to render objects at different distances.

For very close objects (e.g. table-top AR), you may need to reduce the near clip plane. For distant objects, increase the far clip plane. Access these through the Filament `Camera` entity available in the scene.
